gdbstub_arch/src/riscv/mod.rs

```diff
@@ -15,21 +15,108 @@ pub enum Riscv64 {}
 impl Arch for Riscv32 {
     type Usize = u32;
     type Registers = reg::RiscvCoreRegs<u32>;
-    type RegId = reg::id::RiscvRegId<u32>;
     type BreakpointKind = usize;
+    type RegId = reg::id::RiscvRegId<u32>;
```

Comment:
any reason this field moved? I don't really mind either way

Reply:
My IDE (Clion) suggested to swap them to match the base definition. Happy to revert if you prefer

Reply:
I think its OK